configs/scnet/README.md

```diff
@@ -0,0 +1,47 @@
+# SCNet
+
```

Comment:
For algorithms, now the readme requires to add a tag [ALGORITHM] for calculating algorithm statistics. See example [here](https://github.com/open-mmlab/mmdetection/tree/master/configs/tridentnet).